.gitignore

```diff
@@ -24,3 +24,5 @@ onigumo-*.tar
 
 # Temporary files, for example, from tests.
 /tmp/
+
+onigumo
```

Comment:
This is not relevant to the issue. If it was, it’d be nice to attach a comment that this is a compiled binary.

Reply:
OK